### PR TITLE
Redesign search results page

### LINKED ISSUE #1381: Design new dashboard search result page
Current search produces results without clear division of result type and it is unclear on first sight if item is product, order or user.

We might want to make results divided into named easier to navigate separate sections.

Dashboard search should enable quick access to top result so it might be a good idea to limit results to only few items and provide detailed info for each item, ex. order customer info, user first and last name, product description preview.

### What I'm trying to achieve

Make dashboard result page easier to navigate

### Steps to reproduce the problem

1. Go to dashboard search
2. Type search that results in long, multi-type result list

### What I expected to happen

Results formatted in easy to comprehend, more readable way

### What happened instead/how it failed

result

### PR DESCRIPTION
**Pull request by dominik-zeglen**
Resolves #1381 

zrzut ekranu 2017-12-08 o 10 33 23

localhost_8000_dashboard_search__q jones iphone 5 1
localhost_8000_dashboard_search__q jones iphone 5

### CODE REVIEW

**Comment by patrys** [comment]
How many results do we show? #1381 mentioned showing top N results with a "show more" link.

**Comment by dominik-zeglen** [comment]
ATM we show only those N first (defined somewhere in settings) results, because showing all is kinda problematic (summon @salwator @elwoodxblues )

**Comment by elwoodxblues** [comment]
@patrys We show top 5 results now. Although "Show more" appears on the design, we didn't want to add it now as it requires a bit more work than we initially thought.

File: templates/dashboard/search/results.html [hunk 1]
```diff
@@ -16,99 +16,93 @@
...
+                    {% gross product.price html=True %}
+                  </small>
+                </p>
+              </a>
+            </li>
+          {% endfor %}
+        </ul>
+      {% endif %}
+      {% if orders %}
+        <h6>Orders</h6>
```
Comment (elwoodxblues) [inline templates/dashboard/search/results.html]: Missing translation

File: templates/dashboard/search/results.html [hunk 2]
```diff
@@ -16,99 +16,93 @@
...
+                    {% endblocktrans %}
+                  </small>
+                </p>
+              </a>
+            </li>
+          {% endfor %}
+        </ul>
+      {% endif %}
+      {% if users %}
+        <h6>Users</h6>
```
Comment (elwoodxblues) [inline templates/dashboard/search/results.html]: Missing translation.

Comment (dominik-zeglen) [reply templates/dashboard/search/results.html]: Good point.

**Pull request merged**